crates/forge/src/main.rs

```diff
@@ -63,6 +63,11 @@ fn main_execution() -> Result<()> {
             String::from_utf8_lossy(&build_output.stdout)
         )
     }
+    let scarb_stdout = String::from_utf8_lossy(&build_output.stdout);
+
+    if scarb_stdout.contains("warn:") {
+        eprintln!("{scarb_stdout}");
+    }
```

Comment:
I wonder if we should only print warnings or just redirect all stdout and stderr. With this solution, we'll risking hiding some errors in case rust output format changes.

Reply:
I changed it. In the current version, Scarb inherits stdout, so we will write everything